doc/install.md

```diff
@@ -0,0 +1,145 @@
+be run on any Linux box) and a server (which is intended to be run in a place with good uplink connectivity). If 
+you are interested in running your own station, you most likely want to deploy just the station and use existing
+server. Please contact someone from the satnogs team and we'll hook you up.
+
+# Station installation
+
+You need to install [Rasbian](https://www.raspbian.org/) on your Raspberry Pi. Please follow any Raspbian installation
+instruction, such as [this one](https://www.raspberrypi.org/documentation/installation/installing-images/). Once
+done, connect to your Pi and do the following as root:
+
+1. **Install necessary dependencies**:
+
+```
+apt update
+apt install python3-minimal git rtl-sdr sox imagemagick
+```
+
+Also, the following tools are needed:
+
+1. noaa-apt - download Raspberry Pi binaries from https://noaa-apt.mbernardi.com.ar/download.html
+2. medet - https://github.com/artlav/meteor_decoder, binaries available from: http://orbides.org/page.php?id=1023
+3. meteor-demod - https://github.com/artlav/meteor_decoder
+
+2. **Create satnogs user**:
+
+```
+# useradd satnogs
+```
+
+3. Now **switch to the satnogs user**:
+```
+su - satnogs
+```
+
+4. **Get the latest satnogs-gut code**. This and following steps should be done as satnogs user.
+
+```
+git clone https://github.com/gut-space/satnogs
+```
+
+5. **Install python dependencies**:
+
+```
+cd satnogs/station
+python3 setup.py install
+```
+
+This step will install necessary dependencies. It is a good practice to install them in virtual environment. However,
+since the scripts will be called using crontab, it would've complicated the setup.
+
+5. **Run the initial setup** script:
+
+```
+python3 station/setup.py
+```
+this script will conduct several things, such as setting up crontab to run observations periodically, create config file if there isn't any etc.
+
+# 6. Station management
+
+There is a command line tool used to manage the station. You can run it with:
+
+```
+station/cli.py
+
+usage: satnogs-gut [-h] {clear,plan,config} ...
+
+positional arguments:
+  {clear,plan,config}  commands
+    clear              Clear all schedule receiving
+    plan               Schedule planning receiving
+    config             Configuration
+
+optional arguments:
+  -h, --help           show this help message and exit
+  
+```
+
+You can use it to inspect your configuration, clear or schedule upcoming transmissions.
+
+7. **Tweak your config**. 
+
+Run the cli command first:
+```
+python station/cli.py
+```
+
+It will create a tempate config for you. The config file is stored in ~/.config/satnogs-gut/config.yml. The recommended way of tweaking it is to use the cli.py command itself. For example, to set up your location, you can do:
+
+```
+python station/cli.py config location -lat 54.34 -lng 23.23 -ele 154
+```
+
+8. **Schedule observations**
+
+Once your station coordinates are set, you can tell satnogs to schedule observations:
+
+```
+python station/cli.py plan
```

Comment:
I think that here should be information about --force flag. Call station/cli.py set plan job in Crontab. It is called on 4.00 AM by default. User may be confused that he doesn't see any planned job after call it. Maybe observations should be planned when planning job is added. If you agree, please add issue for it.